src/components/capture/creation/index.tsx

```diff
@@ -94,7 +94,7 @@ function NewCaptureModal() {
     );
     const specFormData = useCaptureCreationStore(selectors.specFormData);
     const disoverLink = useCaptureCreationStore(selectors.disoverLink);
```

Comment:
There is a typo in the variable name `disoverLink`. The _c_ is missing in the word _discover_.